lte/gateway/c/core/oai/test/amf/test_amf.cpp

```diff
@@ -10,11 +10,15 @@
  */
 
 #include "util_nas5g_pkt.h"
-#include "dynamic_memory_check.h"
 #include <gtest/gtest.h>
+#include "intertask_interface.h"
+#include "../../tasks/amf/amf_app_ue_context_and_proc.h"
+
 
 using ::testing::Test;
 
+task_zmq_ctx_t grpc_service_task_zmq_ctx;
```

Comment:
is this used? If not remove it.

Reply:
n11 has a dependency with grpc_service, it requires  grpc_service_task_zmq_ctx, If I remove the above, getting the below error due to dependency.
![image](https://user-images.githubusercontent.com/83060027/128896577-9e46328e-f9b7-40a1-8f86-49bdd2f5a77f.png)